.github/workflows/ci_pull_request_open.yml

```diff
@@ -0,0 +1,62 @@
+name: Pull Request Open
+on:
+  pull_request:
+jobs:
+  ci-build-test-pr:
+    runs-on: ubuntu-latest
+    env:
+      MAVEN_OPTS: "-Xmx1500m -XX:+HeapDumpOnOutOfMemoryError"
+      GH_TOKEN: ${{ github.token }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Set up Maven
```

Comment:
If we use `mvnw` wrapper included in the repository there's no need to explicitly configure Maven and we can remove this step.